### ISSUE TITLE
Make sure we pass the board 

### ISSUE THREAD

**Issue by shawntanzk** [obophenotype/provisional_cell_ontology#16]
see here: 
https://obofoundry.org/obo-nor.github.io/dashboard/index.html 

we will be added to the provisional board after application, but might be good to look through beforehand anyway

**Comment by shawntanzk** [comment]
@dosumis would be good if we could make the github submission request asap so we can be placed on the provisional board and fix anything that we need to before acceptance

**Comment by dosumis** [comment]
How can we check before?  Is this all folded into ROBOT verify?

**Comment by matentzn** [comment]
I can run it locally for you if you are quick.

**Comment by shawntanzk** [comment]
I don't think we are ready yet @matentzn - need to fix some stuff at the BDSO side before we can pass this. Just thought we could get it on the board soon to diagnose any problems we might have missed too. We can go through the normal way of applying once we handle the issues in BDSO side :) thanks heaps!

**Comment by scheuerm** [comment]
Can you explain what you mean by "make sure we pass the board"?

**Comment by shawntanzk** [comment]
@scheuerm currently to pass checks to enter the OBO foundry, we need to pass a few checks. These checks are represented by this board: https://obofoundry.org/obo-nor.github.io/dashboard/index.html 
Currently we definitely cannot pass it as we have a few issues so I'm holding off submitting to the foundry till we at least fix stuff in ROBOT report which we can run locally first

**Comment by shawntanzk** [comment]
recent release showed up with no errors - will wait till we are on the board with no errors to close ticket

**Comment by shawntanzk** [comment]
passed
